utils/summaries.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, directory):
         self.directory = directory
 
     def create_summary(self):
-        writer = SummaryWriter(logdir=os.path.join(self.directory))
+        writer = SummaryWriter(log_dir=os.path.join(self.directory))
```

Comment:
in my version this is logdir ?

Reply:
in fact even in my own documentation for 1.1 it says you are correct, but In practice I see logdir